src/main/kotlin/lotto/Controller.kt

```diff
@@ -0,0 +1,10 @@
+package lotto
+
+fun main() {
+    val price = InputNumber.buy()
+    val buyTicket = Ticket().buyTickets(price)
+    val lotto = Ticket().apply { tickets(buyTicket) }
+    ResultView.printBuyedLottoTicket(lotto.purchasedLotto)
```

Comment:
계속 Ticket() 인스턴스를 새로 생성하고 계신것 같아요.

```suggestion
    val price = InputNumber.buy()
    val ticket = Ticket()

    val lottoCount = ticket.buyTickets(price)
    ticket.tickets(lottoCount)
    ResultView.printBuyedLottoTicket(ticket.purchasedLotto)
```

`Ticket()` 시 Ticket 인스턴스가 생성이 되며, 생성된 인스턴스를 이용해 수행하셔야 할 것 같습니다.